tests/functional/adapter/test_snapshot.py

```diff
@@ -0,0 +1,204 @@
+from typing import Dict, Iterable
+
+import pytest
+
+from dbt.tests.util import run_dbt
+
+from tests.functional.adapter.common import (
+    get_records,
+    update_records,
+    insert_records,
+    delete_records,
+    clone_table,
+    add_column
+)
+
+
+_SEED = """
```

Comment:
I think the rule we've been following is if these constants are > 50 lines we've been moving them to a separate fixture.py file

Reply:
This is 31, but I can do that if you want.

Reply:
Was including all the constants, up to you whether it's better for readability

Reply:
I moved this into a separate file.